platforms/corda-ent/charts/nmap/templates/deployment.yaml

```diff
@@ -17,7 +17,6 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: {{ .Values.nodeName }}
```

Comment:
matchLabels must match metadata.labels

Reply:
Fixed in commit [7568f6c](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/844/commits/7568f6cb9b32e8f50b7d860d97e999f40da288eb)